### DISCUSSION TITLE
Organising...?

### DISCUSSION THREAD

**Discussion by xvaara** [cdmoro/bootstrap-vue-3#173] [General]
Hey, seems that people are finding this project. That's great. I'm not sure what @cdmoro has planned for the project.

Last spring I register github organisation vue-bootstrap: https://github.com/vue-bootstrap that we could use. Also created a discord https://discord.gg/NMKAJHGJ to chat. But I kinda lost my motivation to organise at the moment due to other vue3 related projects stalled.

Would the project like to use those?

**Comment by cdmoro** [comment]
Hello there, just in case I created a github organization as well: https://github.com/bootstrap-vue-3 
We could transfer this project to that organization, which has the same name (like [BoostrapVue organization](https://github.com/bootstrap-vue)).

**Reply by xvaara** [reply]
I always disliked the fact that it's bootstrapVue, not vue-bootstrap like every other vue project. And of course it depends on that is your focus to be the same for vue3 as BootstrapVue or the future of bootstrap on vue.

oh, And I also registered the vue-bootstrap npm organisation.

**Reply by micene09** [reply]
Totally agree on naming, in another discussion i proposed a merge to the BootstrapVue organization...but in alternative, vue-bootstrap is definitely a more Vue standard lib name.
Also, notice the naming of other libraries like ng-bootstrap and react-bootstrap.
